### PR TITLE
Story/ccls 1720 cancel client registration

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**
Add the ability to cancel client registration.

Fixed minor issues with error messages.
Fixed issues with git packages builds

### CODE REVIEW

File: src/main/resources/templates/application/client/basic-client-details.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <div id="main-content">
         <form action="#" th:action="@{/application/client/details/basic}" th:object="${clientDetails}" method="post">
             <div th:replace="~{partials/error-summary :: error-summary}"></div>
-            <span class="govuk-caption-l">About your application</span>
+            <span class="govuk-caption-l">About your client</span>
             <h1 class="govuk-heading-l">
                 Basic Client Details
```
Comment (keith-h) [inline src/main/resources/templates/application/client/basic-client-details.html:16]: Should this be sentence case? `Basic client details`?

Comment (PhilDigitalJustice) [reply src/main/resources/templates/application/client/basic-client-details.html:16]: Yeah will amend

File: src/main/resources/templates/application/select-office.html [hunk 2]
```diff
@@ -17,12 +17,12 @@ <h1 class="govuk-heading-l">
                 <div th:replace="~{partials/error-summary :: error-summary}"></div>
               <div class="govuk-body-l">
                   Please select the office where the application will be held. Alternatively, click
-                  <a href="/application/copy-case-search">Copy Case </a>
+                  <a href="/application/copy-case/search">Copy Case </a>
```
Comment (keith-h) [inline src/main/resources/templates/application/select-office.html:20]: Sentence case? `Copy case`?

File: src/main/resources/templates/application/client/basic-client-details.html (hunk 1 above)
Comment (PhilDigitalJustice) [reply src/main/resources/templates/application/client/basic-client-details.html]: amended

**Pull request merged**